### ISSUE TITLE
Celluloid won't run native on wayland (sway), runs fine in Xwayland

### ISSUE THREAD

**Issue by oniGino** [celluloid-player/celluloid#681]
running celluloid in my shell won't work alone
```
celluloid

(io.github.celluloid_player.Celluloid:45158): Gdk-WARNING **: 11:44:52.164: Failed to load cursor theme Adwaita

(io.github.celluloid_player.Celluloid:45158): celluloid-WARNING **: 11:44:52.261: Failed to guess content type for mount Filesystem root: mount doesn’t implement content type guessing

(io.github.celluloid_player.Celluloid:45158): celluloid-WARNING **: 11:44:52.261: Failed to guess content type for mount Windows: mount doesn’t implement content type guessing

(io.github.celluloid_player.Celluloid:45158): celluloid-WARNING **: 11:44:52.261: Failed to guess content type for mount tmp: mount doesn’t implement content type guessing

(io.github.celluloid_player.Celluloid:45158): celluloid-WARNING **: 11:44:52.261: Failed to guess content type for mount portage: mount doesn’t implement content type guessing
VA error: wayland: Wayland roundtrip error: Protocol error (errno 71)
Gdk-Message: 11:44:52.302: Error 71 (Protocol error) dispatching to Wayland display.
```

however if I pass a --display that app runs without issue in Xwayland

```
 ~  celluloid --display=:0

(io.github.celluloid_player.Celluloid:45979): celluloid-WARNING **: 11:49:20.720: Failed to guess content type for mount Filesystem root: mount doesn’t implement content type guessing

(io.github.celluloid_player.Celluloid:45979): celluloid-WARNING **: 11:49:20.720: Failed to guess content type for mount Windows: mount doesn’t implement content type guessing

(io.github.celluloid_player.Celluloid:45979): celluloid-WARNING **: 11:49:20.720: Failed to guess content type for mount tmp: mount doesn’t implement content type guessing

(io.github.celluloid_player.Celluloid:45979): celluloid-WARNING **: 11:49:20.720: Failed to guess content type for mount portage: mount doesn’t implement content type guessing
 ```

also if I run `mpv` from `smplayer` or directly from the shell it runs without issue. It seems that celluloid is doing something strange here.

Am I missing something?

**Comment by gnome-mpv** [comment]
I can't reproduce the issue. There's a delay before the window shows up, but running `systemctl --user import-environment` fixes the problem. I tested with Sway 1.6.1.

Did you try running other GTK apps? Maybe there's a problem with GTK in your setup.

Also, are you using Flatpak?

**Comment by oniGino** [comment]
sorry its gentoo  and version 0.21 no flatpak

other gtk apps seems fine, ita weird to me that i can run mpv directly without issue though

ill try that systemctl user import and see if that helps

**Comment by oniGino** [comment]
thanks you are correct my issue was a broken release of libva-2.10.0 compatibility with XWayland, its fixed in libva-2.12.0
